src/components/Sidebar.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { FaRegListAlt, FaUserFriends, FaUserTie, FaRegEnvelope, FaCog, FaBars } from 'react-icons/fa';
+import { FiLogOut } from 'react-icons/fi';
+import { Link } from 'react-router-dom';
+import { RxDashboard } from 'react-icons/rx';
+
+export default function Sidebar() {
+  const [activeLink, setActiveLink] = useState('Dashboard');
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleLinkClick = (link: string) => {
+    setActiveLink(link);
+  };
+
+  const toggleSidebar = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const linkClasses = (link: string) =>
```

Comment:
You can ommit this function and use the `cn` function that we have defined in `src/utils` , directly in the classnames